dali/imgcodec/decoders/nvjpeg_lossless/nvjpeg_lossless.cc

```diff
@@ -33,12 +33,24 @@ NvJpegLosslessDecoderInstance::NvJpegLosslessDecoderInstance(
       event_(CUDAEvent::Create(device_id)) {
   DeviceGuard dg(device_id_);
   // TODO(janton): use custom allocators (?)
-  CUDA_CALL(nvjpegCreateEx(NVJPEG_BACKEND_LOSSLESS_JPEG, NULL, NULL, 0, &nvjpeg_handle_));
+  auto ret = nvjpegCreateEx(NVJPEG_BACKEND_LOSSLESS_JPEG, NULL, NULL, 0, &nvjpeg_handle_);
+  if (ret != NVJPEG_STATUS_SUCCESS) {
+    // some nvJPEG version doesn't support NVJPEG_BACKEND_LOSSLESS_JPEG so disable it if
+    // it failed to initialize
+    DALI_WARN_ONCE("The available nvJPEG library version doesn't support Lossless format, please "
+                    " update to the latest one.");
```

Comment:
```suggestion
                    "update to the latest one.");
```
You already have a space in the previous line

Reply:
Fixed